.github/Dockerfile-release.arm32v6

```diff
@@ -1,10 +1,15 @@
+# hadolint ignore:DL3007
 FROM busybox:latest as build
 RUN touch /config.yaml
 
+# hadolint ignore:DL3006
 FROM arm32v7/alpine
 ENV AUTOCERT_DIR /data/autocert
 WORKDIR /pomerium
-COPY --from=multiarch/qemu-user-static /usr/bin/qemu-aarch64-static /usr/bin/
+# hadolint ignore:DL3006
+FROM multiarch/qemu-user-static as qemu-user-static
```

Comment:
Should this be above `FROM arm32v7/alpine`?  My understanding is the current ordering would use `multiarch/qemu-user-static` is the base of the final image, which I don't think we want.